test/contracts/consumer/fabric8auth/api_status.go

```diff
@@ -0,0 +1,55 @@
+package fabric8auth
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-auth/test/contracts/model"
+	"github.com/pact-foundation/pact-go/dsl"
+)
+
+// AuthAPIStatus defines contract of /api/status endpoint
+func AuthAPIStatus(t *testing.T, pact *dsl.Pact) {
+
+	log.Printf("Invoking AuthAPIStatus now\n")
+
+	// Pass in test case
+	var test = func() error {
+		u := fmt.Sprintf("http://localhost:%d/api/status", pact.Server.Port)
+		req, err := http.NewRequest("GET", u, nil)
```

Comment:
Please check for `err != nil` before setting something on the `req` object.

Reply:
Thanks for the tip - fixed.